FreeRTOS/FreeRTOS
# Remove deprecated macro (ipconfigRAND32) references

Description
-----------
ipconfigRAND32/ipconfigRAND64 macros were deprecated/removed from the FreeRTOS+TCp library. However, some demos still call the macros to get random numbers.

This PR removes the references of those macros from the demo sources.

Test Steps
-----------


Related Issue
-----------
https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/issues/413


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
